e2e/wdio/driver/wdio.ts

```diff
@@ -175,7 +175,7 @@ export function clearValue(selector: string, index: number = 0, waitTime = defau
 export function getElementSize(selector: string, index?: number):  WebdriverIO.SizeReturn;
 export function getElementSize(selector: string, index: number, prop: 'width' | 'height'): number;
 export function getElementSize(selector: string, index: number = 0, prop?: 'width' | 'height'): number | WebdriverIO.SizeReturn {
-    return prop ? $$(selector)[index].getSize() : $$(selector)[index].getSize(prop);
+    return prop ? $$(selector)[index].getSize(prop) : $$(selector)[index].getSize();
```

Comment:
`return $$(selector)[index].getSize(propa || void 0)` should work the same way

Reply:
fixed.